docs/developer-docs/build/languages/motoko/control-flow.md

```diff
@@ -71,17 +71,17 @@ let address = label exit : ?(Text, Host) {
 }
 ```
 
-Naturally, labeled common expressions don’t allow `continue`. In terms of typing, both `⟨expr⟩` and `⟨alt-expr⟩`​'s types must conform with the label’s declared `⟨type⟩`. If a label is only given a `⟨name⟩`, then its `⟨type⟩` defaults to unit (`()`). Similarly a `break` without an `⟨alt-expr⟩` is shorthand for the value unit (`()`).
+Naturally, labeled common expressions don’t allow `continue`. In terms of typing, both `<expr>` and `<alt-expr>`​'s types must conform with the label’s declared `<type>`. If a label is only given a `<name>`, then its `<type>` defaults to unit (`()`). Similarly a `break` without an `<alt-expr>` is shorthand for the value unit (`()`).
 
 ## Option blocks and null breaks
 
-Like many other high-level languages, Motoko lets you opt in to `null` values, tracking possible occurrences of `null` values using option types of the form `?T`. This is to both to encourage you to avoid using `null` values when possible, and to consider the possibility of `null` values when necessary.
+Like many other high-level languages, Motoko lets you opt in to `null` values, tracking possible occurences of `null` values using option types of the form `?T`. This is to both to encourage you to avoid using `null` values when possible, and to consider the possiblity of `null` values when necessary.
```

Comment:
```suggestion
Like many other high-level languages, Motoko lets you opt in to `null` values, tracking possible occurrences of `null` values using option types of the form `?T`. This is to both to encourage you to avoid using `null` values when possible, and to consider the possibility of `null` values when necessary.
```
Occurrence: https://en.wiktionary.org/wiki/occurence